os/include/tinyara/pm/pm.h

```diff
@@ -633,7 +633,28 @@ enum pm_state_e pm_querystate(int domain);
  *   Incorrect frequency scaling level given as input will lead to an assert.
  *
  ****************************************************************************/
+#ifdef CONFIG_PM_DVFS
```

Comment:
It would be better if the conditional flags also includes the documentation.

Reply:
done~